### DISCUSSION TITLE
can you add a --exclude-dir flag

### DISCUSSION THREAD

**Discussion by abdulrahim2002** [BurntSushi/ripgrep#2837] [General] [answered]
hi team,
- can you add a `--exclude-dir` flag to prevent rg from searching that directory.
- when i run rg in `/` it goes into `/sys/*`  and `/proc/*` for which it generates alot of stuff
- maybe you can do it with .ignore files or blob something. but adding a flag is much simpler for end user. 
- Thanks

**Comment by BurntSushi** [comment, marked as answer]
There is a flag. `-g '!/sys`.
